docs/containers-transports.5.md

```diff
@@ -0,0 +1,53 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+# NAME
+containers-transports - description of supported transports for copying container images
+
+# DESCRIPTION
```

Comment:
Double hash this please `##`

Reply:
Done.

Reply:
I don't see it?

Reply:
Actually, shouldn’t the standard man-pages(7) (NAME, DESCRIPTION, SEE ALSO, AUTHORS) section headers all be at the same level?  https://github.com/cpuguy83/go-md2man/blob/master/go-md2man.1.md uses a single has sign.

Reply:
Yes It should be
```
## NAME
and
## DESCRIPTION
```